### DISCUSSION TITLE
[blocks]: Sign up with OAuth providers (e.g. Google, GitHub)

### DISCUSSION THREAD

**Discussion by andreibadescu** [shadcn-ui/ui#5377] [Blocks Request]
### Description

Everyone adds this and is a pain to do every time for every project.

### Example

any website with google sign up

**Comment by cookieMonsterDev** [comment]
What is the point? Shadcn just gives you the patterns of components, it is totally up to you to decide how to build auth. As example you can use [Auth.js](https://authjs.dev/) it has inbuilt Google and GitHub providers.

**Comment by HeyClos** [comment]
Leaving this here for reference.
Shadcn Log in Component (No functionality) :
https://github.com/huntabyte/shadcn-svelte/blob/next/sites/docs/src/lib/registry/new-york/block/login-01/components/login-form.svelte
